guestbook/README.md

```diff
@@ -50,24 +50,23 @@ The manifest file, included below, specifies a Deployment controller that runs a
 1. Launch a terminal window in the directory you downloaded the manifest files.
 2. Apply the Redis Master Deployment from the `redis-master-deployment.yaml` file:
 
-        kubectl apply -f redis-master-deployment.yaml
+       kubectl apply -f redis-master-deployment.yaml
         
-{% include code.html language="yaml" file="redis-master-deployment.yaml" ghlink="/docs/tutorials/docs/tutorials/stateless-application/redis-master-deployment.yaml" %}
+   {% include code.html language="yaml" file="redis-master-deployment.yaml" ghlink="/docs/tutorials/docs/tutorials/stateless-application/redis-master-deployment.yaml" %}
```

Comment:
- /docs/tutorials/docs/tutorials/ is wrong
- can you please update the URL as if we created another directory, i.e. guestbook/ under stateless-deployment/?